kuma/users/jinja2/users/ban_user_and_cleanup.html

```diff
@@ -0,0 +1,67 @@
+{% extends "base.html" %}
+
+{% block body_attributes %}{% endblock %}
+
+{% block bodyclass %}user ban-user{% endblock %}
+
+{% block title %}{{ _('Ban') }} {{ detail_user }}{% endblock %}
```

Comment:
The translation needs to be full-phrase translation with interpolation, like:

    {% block title %}{{ _('Ban %(user)s', user=detail_user) }}{% endblock %}